include/avif/avif.h

```diff
@@ -140,7 +140,8 @@ typedef enum avifResult
     AVIF_RESULT_IO_ERROR,
     AVIF_RESULT_WAITING_ON_IO, // similar to EAGAIN/EWOULDBLOCK, this means the avifIO doesn't have necessary data available yet
     AVIF_RESULT_INVALID_ARGUMENT, // an argument passed into this function is invalid
-    AVIF_RESULT_NOT_IMPLEMENTED   // a requested code path is not (yet) implemented
+    AVIF_RESULT_NOT_IMPLEMENTED,  // a requested code path is not (yet) implemented
+    AVIF_RESULT_NEED_POST_PROCESS // a requested code path need post process
```

Comment:
1. Nit: In the comment, change "need" to "needs". (The enum name can say "NEED".)

Also, it is more common to say "post-processing". "Post-process" sounds like a verb.

2. This avifResult value is only used by libavif internally. Ideally it should not be in the public API. I don't have a good solution right now, other than adding an internal enum for the return value of avifImageYUVToRGBLibYUV() and similar functions.